addons/docs/package.json

```diff
@@ -130,6 +132,7 @@
     "babel-loader": "^8.0.0",
     "react": "^16.8.0 || ^17.0.0",
     "react-dom": "^16.8.0 || ^17.0.0",
+    "svelte": "^3.31.2",
```

Comment:
needs to be an optional peer dependency also. hopefully we can refactor this soon cc @phated